docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -191,6 +191,83 @@ For the most accurate migration assessment, the source database must be actively
     - [Live migration with fall-forward](../../migrate/live-fall-forward/)
     - [Live migration with fall-back](../../migrate/live-fall-back/)
 
+## Bulk assessment
+
+The Bulk Assessment command (`access-migration-bulk`) allows you assess multiple schemas in one or more database instances simultaneously. Bulk assessment enables the following:
+
+- Multi-schema assessment - Assess multiple schemas across one or more database instances with a single command, streamlining the migration planning process.
+- Centralized reporting - All assessment reports are generated and stored in a single organized directory, so you can access and review the migration assessments for all schemas in one place.
+
+![Oracle bulk assessment](/images/migrate/assess-migration-bulk.png)
+
+### Prerequisites
+
+Prepare source databases as described in [Generate a migration assessment](#generate-a-migration-assessment-report).
```

Comment:
How about:
```
Prepare source databases for each schema....
```

since user needs to do that step on database for each schema separately.